### ISSUE TITLE
Color contrast: invert button style

### ISSUE THREAD

**Issue by gtvj** [nationalarchives/tna-frontend-toolkit#14]
The invert button style does not have sufficient color contrast. We need to come up with an inverted style that meets WCAG.
